packages/@ember/-internals/metal/lib/tracked/debugging.ts

```diff
@@ -0,0 +1,110 @@
+import { Tag, CONSTANT_TAG, UpdatableTag } from '@glimmer/reference';
+
+import { Tracker } from './tracker';
+
+type Option<T> = T | null;
+
+Ember.EMBER_DEBUG = {};
+
+let TAG_MAP = new Map<Tag | UpdatableTag, any>();
```

Comment:
I was thinking about storing a map of tags to their host objects in here.
I later found out that none of the invocations to consume (or other places where `CURRENT_TRACKER.add` is called) contain a references to the object/instance. :(